Packs/SnortIPBlocklistFeed/Integrations/SnortIPBlocklist/SnortIPBlocklist.py

```diff
@@ -0,0 +1,218 @@
+from typing import Dict, List, Optional
+
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    The _http_request() has to allow redirects to support the snort IP blocklist
+    """
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+
+        result = []
+
+        res = self._http_request('GET',
+                                 url_suffix='',
+                                 full_url=self._base_url,
+                                 resp_type='text',
+                                 allow_redirects=True
+                                 )
+
+        # In this case the feed output is in text format, so extracting the indicators from the response requires
+        # iterating over it's lines solely. Other feeds could be in other kinds of formats (CSV, MISP, etc.), or might
+        # require additional processing as well.
+        try:
+            indicators = res.splitlines()
+
+            for indicator in indicators:
+                # Infer the type of the indicator using 'auto_detect_indicator_type(indicator)' function
+                # (defined in CommonServerPython).
+                if indicator_type := auto_detect_indicator_type(indicator):
+
+                    result.append({
+                        'value': indicator,
+                        'type': indicator_type,
+                        'FeedURL': self._base_url
+                    })
+
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data as indicator. \n\nError massage: {err}')
+        return result
+
+
+def test_module(client: Client) -> str:
+    """Builds the iterator to check that the feed is accessible.
+    Args:
+        client: Client object.
+    Returns:
+        Outputs.
+    """
+    client.build_iterator()
```

Comment:
I think it would be better to test a limited fetch here (I will add this correction also to the HelloWorldFeed).
```suggestion
    fetch_indicators(client=client, limit=1)
```